docs/APIReference-SelectionState.md

```diff
@@ -47,6 +47,10 @@ For instance, when extracting a slice of text from a block based on a
 `SelectionState`, it is irrelevant whether the selection is backward:
 
 ```
+var selection = editorState.getSelection();
```

Comment:
I would call this 'selectionState', and then we can assume that it's the same variable being used below.

I'm also wondering if it would be more clear to rename 'myContentBlock' to 'currentContentBlock'?